unlock-app/src/components/interface/checkout/CryptoCheckout.tsx

```diff
@@ -137,13 +167,41 @@ export const CryptoCheckout = ({
         hasOptimisticKey={hasOptimisticKey}
         purchasePending={purchasePending}
       />
+      <SmallButton onClick={() => setIsAdvanced(!isAdvanced)}>
+        {isAdvanced ? 'Close advanced' : 'Advanced'}
+      </SmallButton>
+      {isAdvanced && (
+        <>
+          <Label>Recipient</Label>
+          <Input
+            type="text"
+            placeholder="Recipient address"
+            name="recipient"
+            onChange={onRecipientChange}
+            style={{ marginBottom: '0.2rem' }}
+          />
+          {recipient?.length ? (
+            recipientValid ? (
```

Comment:
I am not sure we nee to show this message. Let's assume that this is valid unless it is marked as invalid.